README.md

```diff
@@ -49,17 +48,19 @@ Once [commercetools HTTP API Extensions](https://docs.commercetools.com/http-api
 - Follow [Integration Guide](./extension/docs/WebComponentsIntegrationGuide.md) for information how to integrate your shop with this module.
 - Follow [How to run](extension/docs/HowToRun.md) the extension module.
 
-## Notification module 
+### Notification module 
 
 [![Docker Pulls](https://img.shields.io/docker/pulls/commercetools/commercetools-adyen-integration-notification)](https://hub.docker.com/r/commercetools/commercetools-adyen-integration-notification)
 
-Notification module is a publicly exposed service which receives asynchronous notifications sent by Adyen, 
+Notification module is a publicly exposed service which receives asynchronous notifications sent by Adyen,
 Through notifications, Adyen provides asynchronously payment status changes like authorization, charge, or refund of the payment.
-The notification module will process the notification and update the matching commercetools payment accordingly.
+The notification module will process the notification sent by Adyen and matches the commercetools payment for this notification, and modifies commercetools payment accordingly.
```

Comment:
In the task you have this from Stefan: `It doesn’t hurt to repeat that for the Notification Module. I would explicitly mention that only the Payment object is updated, but never the Cart or the Order`

Maybe also add this statement here explicitly.

Reply:
Already added a note to overview section, on line 36 https://github.com/commercetools/commercetools-sync-java/commit/52a8073a6c120a615e39610b3d31ddf373808db8